zitadel/local/README.md

```diff
@@ -89,40 +92,73 @@ scripts. What we need to pay attention to completing as below:
 
 ```shell
 terraform init
-terraform plan -var-file=environments/local.tfvars -var-file=environments/environment.tfvars
-terraform apply -auto-approve -var-file=environments/local.tfvars -var-file=environments/environment.tfvars
+terraform plan -var-file=environments/local.tfvars
+terraform apply -auto-approve -var-file=environments/local.tfvars
 ```
 
 After successfully executing the Terraform script, the Terraform outputs is as below:
-![terraform-output-config-items](compose/statics/terraform-outputs-config.png)
+![view_terraform_outputs](statics/view_terraform_outputs.png)
 
 The default IAM admin user login the console of Zitadel to view added configurations:
-![view-configs-on-console](compose/statics/view-configs-on-console.png)
+![view_org_project_apps](statics/view_org_project_apps.png)
 
-### Add Admin User of Xpanse
+## Add Users for Xpanse
+
+### Manual Configuration on Console
```

Comment:
this topic does not belong to `Add Users for Xpanse` topic. 
Also this topic is needed due to a bug in the provider. We can document that too. 
https://github.com/zitadel/terraform-provider-zitadel/issues/75